client/gutenboarding/gutenboard.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * WordPress dependencies
+ */
+import '@wordpress/editor'; // This shouldn't be necessary
+
+import React, { useState } from 'react';
+import {
+	BlockEditorKeyboardShortcuts,
+	BlockEditorProvider,
+	BlockList,
+	WritingFlow,
+	ObserveTyping,
+} from '@wordpress/block-editor';
+import { Popover, SlotFillProvider, DropZoneProvider } from '@wordpress/components';
+import { registerCoreBlocks } from '@wordpress/block-library';
+import '@wordpress/format-library';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+/* eslint-disable no-restricted-syntax */
```

Comment:
Not sure whether these are necessary, I believe the disables are copied from Gutenberg repo.

Reply:
yep they are in the playground

Reply:
Removed.